client/src/components/Links/LinkItem/LinkItem.jsx

```diff
@@ -0,0 +1,76 @@
+// import styles from './style';
+import React from 'react';
+import {
+    FacebookLinkIcon,
+    WebLinKIcon,
+    WhatsappLinkIcon,
+} from '../../../assets/icons';
+import {
+    LinkItemAction,
+    LinkItemGeneral,
+    LinkItemIcon,
+    LinkItemText,
+    LinkItemTitle,
+    LinkItemWrapper,
+} from './style';
+import PropTypes from 'prop-types';
+
+const getLinkIcon = (type) => {
+    switch (type) {
+        case 'whatsapp':
+            return WhatsappLinkIcon;
+        case 'facebook':
+            return FacebookLinkIcon;
+        case 'web':
+            return WebLinKIcon;
+
+        default:
+            return WebLinKIcon;
+    }
+};
+
+const getLinkActionText = (type) => {
+    switch (type) {
+        case 'whatsapp':
```

Comment:
please use the translation hook and file, `useT` and HE_IL, should be both in Hebrew and Arabic